templates/index.html

```diff
@@ -1,29 +1,118 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <meta charset="utf-8">
-    <title>Community website</title>
-  </head>
-  <body>
-    <img src="static/org_logo.png" alt="org logo" height="42" width="42">
-    <ul>
-      <li><a href="gci/">Google Code-in</a>
-      <li><a href="activity/">GitHub activity</a>
-      <li><a href="twitter/">Twitter Feed</a>
-      <li><a href="info.txt">Build info</a>
-      <li><a href="log/">Logs</a>
-      <li><a href="contributors/">Contributors data</a>
-      <li><a href="meta-review/">Meta-review data</a>
-      <li><a href="gamification/">Gamification leaderboard</a>
-      <li><a title="List of all the issues on organization's main repository on which assignee has not shown any activity for more than 2 months."
-        href="static/inactive-issues.json">Inactive Issues</a>
-      <li><a href="openhub/">OpenHub Data</a>
-      <li><a href="model/">Visit imported data</a>
-      <li><a title="List of all the issues on organization main repository on which someone has opened a pull request without getting assigned to it."
-        href="static/unassigned-issues.json">Unassigned issues activity</a>
-    </ul>
-    {% if isTravis %}
-    <small>This website was built automatically using Travis CI. A link to the build can be found <a href="{{ travisLink }}">here</a>.</small>
-    {% endif %}{# if isTravis #}
-  </body>
-</html>
+{% extends 'base.html' %}
+{% load staticfiles %}
+{% block title %}
+  Community | Homepage
+{% endblock %}
+
+{% block add_css_files %}
+<link rel="stylesheet" href="{% static 'css/index.css' %}">
+{% endblock %}
+
+{% block add_js_files %}
+<script src="//cdnjs.cloudflare.com/ajax/libs/Chart.js/2.7.1/Chart.bundle.js"></script>
+<script src="{% static 'js/index.js' %}"></script>
+{% endblock %}
+
+{% block main-content %}
+
+  <div class="community-header-image">
+    <img src="{% static 'images/os_community_header.jpg' %}">
```

Comment:
This image looks too large IMO.

Reply:
exported from Photoshop itself for web display! Adjusted the header image height to make it look good on large screen otherwise it would have taken whole visible screen to display that image.


Any suggestions? On how can I make it better.
> image looks too large